src/toil/test/src/jobCacheTest.py

```diff
@@ -684,8 +685,8 @@ def _controlledFailTestFn(job, jobDisk, testDir):
                 Hidden.AbstractCacheTest._requirementsConcur(job, jobDisk, cached)
                 Hidden.AbstractCacheTest._returnFileTestFn(job, jobDisk, cached, testDir, 20)
             else:
-                modifiedJobReqs, cached = Hidden.AbstractCacheTest._returnFileTestFn(job, jobDisk, 0,
-                                                                                     testDir, 20)
+                modifiedJobReqs, cached = Hidden.AbstractCacheTest._returnFileTestFn(job, jobDisk,
```

Comment:
Might want to extract `Hidden.AbstractCacheTest` into a variable so you don't have to wrap the arguments so narrowly.